arch/arm/mach-imx/fiohab.c

```diff
@@ -192,6 +192,8 @@ static int hab_status(void)
 		       HAB_SUCCESS) {
 			if (!is_known_rng_fail_event(event_data, bytes)) {
 				printf("HAB events active error\n");
+				printf("Make sure the SPL is correctly signed \
+					and the board is fused\n");
```

Comment:
I think the end result on console will also contain this large empty space between the lines. You can concatenate by having "...signed " (new line) "and the board is fused\n".

Reply:
The kernel practice recommends not to split long strings at all. This doesn't break full-text searching thru code.